gitpod-start-app.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+gp env GEM_HOME=/workspace/.rvm
```

Comment:
These need to get set again because rvm resets the gemdir to the path for gems in the installed ruby version shim folder. Normally this is fine, but as mentioned up top, there is an issue with Gitpod persisting gems.